ui/dashboard/src/pages/signin/email.tsx

```diff
@@ -0,0 +1,157 @@
+import { FormProvider, SubmitHandler, useForm } from 'react-hook-form';
+import {
+  IconRemoveRedEyeOutlined,
+  IconVisibilityOffOutlined
+} from 'react-icons-material-design';
+import { Link, useNavigate } from 'react-router-dom';
+import { signIn } from '@api/auth';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useAuth } from 'auth';
+import { PAGE_PATH_ROOT } from 'constants/routing';
+import * as yup from 'yup';
+import { SignInForm } from '@types';
+import { IconBackspace } from '@icons';
+import { Button } from 'components/button';
+import Form from 'components/form';
+import Icon from 'components/icon';
+import Input from 'components/input';
+import InputGroup from 'components/input-group';
+import AuthWrapper from './elements/auth-wrapper';
+
+const formSchema = yup.object().shape({
+  email: yup.string().email().required(),
+  password: yup
+    .string()
+    .required()
+    .min(4, 'The password you have provided must have at least 4 characters')
+});
+
+const SignInWithEmail = () => {
+  const { syncSignIn } = useAuth();
+  const navigate = useNavigate();
+
+  const [showPassword, setShowPassword] = useState(false);
+  const [showAuthError, setShowAuthError] = useState(false);
+
+  const form = useForm({
+    resolver: yupResolver(formSchema),
+    defaultValues: {
+      email: '',
+      password: ''
+    }
+  });
+
+  const onSubmit: SubmitHandler<SignInForm> = values => {
+    setShowAuthError(false);
+    return signIn(values)
+      .then(response => {
+        syncSignIn(response.token);
+        navigate(PAGE_PATH_ROOT);
+      })
+      .catch(error => {
+        if (error) {
+          setShowAuthError(true);
+        }
+      });
+  };
+
+  const PasswordAddonAction = () => (
+    <Button
+      type="button"
+      variant="grey"
+      className="text-gray-500 size-6"
+      onClick={() => setShowPassword(!showPassword)}
+    >
+      <Icon
+        icon={
+          showPassword ? IconVisibilityOffOutlined : IconRemoveRedEyeOutlined
+        }
+        size="sm"
+      />
+    </Button>
+  );
+
+  return (
+    <AuthWrapper>
+      <Button
+        variant="secondary-2"
+        onClick={() => navigate(PAGE_PATH_ROOT)}
+        className="p-2 h-auto"
+      >
+        <Icon icon={IconBackspace} size="sm" />
+      </Button>
+      <h1 className="text-gray-900 typo-head-bold-huge mt-8">{`Sign in`}</h1>
+      <p className="text-gray-600 typo-para-medium mt-4">
+        {`To access our Demo site, please sign in using the follow in information.`}
+      </p>
+      <div className="text-gray-600 typo-para-medium mt-6">
+        <p>{`Email: demo@bucketeer.io`}</p>
+        <p>{`Password: demo`}</p>
+      </div>
+
+      {showAuthError && (
+        <p className="text-accent-red-500 typo-para-medium mt-6">
+          {`Wrong email or password. Try again or`}
+          <Link to={PAGE_PATH_ROOT}>
+            <span className="underline ml-1">{`create an account.`}</span>
+          </Link>
```

Comment:
Wrong email or password. Try again or
```suggestion
          {`Invalid email or password.`}
```